src/app/services/baw-api/harvest/harvest.service.ts

```diff
@@ -153,7 +154,7 @@ export class ShallowHarvestsService implements StandardApi<Harvest> {
   ): Observable<Harvest> {
     return this.api
       .httpPatch(shallowEndpoint(model, emptyParam), {
-        [this.harvestKind]: { status },
+        [this.harvestKind]: { status: snakeCase(status) },
```

Comment:
i was thinking that we fixed this? 

but because we're not using toJson  this slipped through.

maybe this should be a call toJson and then we pick a just the key we want from the serialized object after that?

Reply:
or make a helper method like toJson that allows us to pick just one field, and it runs through our converters

Reply:
In this case, we are directly passing the status to this function instead of the model. As such we don't have access to the `toJson` function. Ie:

```typescript
this.stages.transition("complete", () => (this.loading = false));
```

Reply:
This feels wrong and like it has potential for future errors. I don't have a better solution though...